src/components/OperationRow.js

```diff
@@ -32,6 +32,7 @@ type Props = {
   navigation: *,
   multipleAccounts?: boolean,
   isLast: boolean,
+  subOperation?: boolean,
```

Comment:
minor feedback on the naming, i suggest as it's a bool, to not confuse with `Operation` to call it `isSubOperation` or maybe `withBackButton` ? 🤔

Reply:
Agreed, rebased, `isSubOperation` for now, I'm not rebasing again! :trollface: